scenes/cslib_math.txt

```diff
@@ -0,0 +1,173 @@
+*comment PI
+*comment ------------------------------
+*comment Returns an approximation of the constant PI.
+*comment ------------------------------
+*comment	returns: 
+*comment		(number) PI
+
+*label pi
+*set cslib_ret 3.1415926535
+*return
+
+
+*comment SIN
+*comment ------------------------------
+*comment Determines the sine of an angle (in radians).
+*comment ------------------------------
+*comment 	params:
+*comment		p_angle(number): an angle
+*comment	returns: 
+*comment		(number) the sine of the given angle
+
+*label sin
+*params p_angle
+
```

Comment:
We shouldn't leave empty lines inside a subroutine. This might introduce a line in the game rendered text. You can leave an empty comment line if you want more whitespace (which I agree is necessary many times).

Reply:
fixed